server/ai_live_video.go

```diff
@@ -614,8 +614,8 @@ func (s *SlowOrchChecker) GetCount() int {
 func LiveErrorEventSender(ctx context.Context, streamID string, event map[string]string) func(err error) {
 	return func(err error) {
 		GatewayStatus.StoreIfNotExists(streamID, "error", map[string]interface{}{
```

Comment:
Just checking that it's intentional to leave `error` as the key here while changing the `error_message` field?

Reply:
yep, i needed a new field name to wrap the error fields so went with generic `error` and renamed the error message field